examples/docs/object_detection.md

```diff
@@ -43,3 +43,23 @@ Your output should look like the following:
 An output image with bounding box will be saved as build/output/detected-dog_bike_car.png:
 
 ![detected-dogs](img/detected-dog_bike_car.png)
+
+## Run object detection example with other engines
+For objection detection application, other than the default model zoo with the default engine, we can also run it with other engines and model zoo. Here, we demonstrate with a pre-trained *YOLOV5s ONNX* model.
+
+First, the following line needs to be added into the dependencies in `build.gradle`. Otherwise, the ONNX model zoo won't be found.
```

Comment:
Since we added to `build.gradle` we don't need mention this is readme

Reply:
Solved